test/evergreen.yml

```diff
@@ -1501,6 +1501,17 @@ tasks:
         vars:
           checkpoint_args: -t c -n 1000000 -k 5000000 -C cache_size=100MB
 
+  - name: test-unit-ppc
+    commands:
+      - func: "get project"
+      - func: "compile wiredtiger"
+        vars:
+          configure_env_vars: CC=/opt/mongodbtoolchain/v3/bin/gcc CXX=/opt/mongodbtoolchain/v3/bin/g++ CPPFLAGS="-I/usr/local/include" LDFLAGS="-L/usr/local/lib" CFLAGS="-g -fPIC"
```

Comment:
Better to change -g to -ggdb

Reply:
Why we need this CPPFLAGS="-I/usr/local/include"?

Reply:
Fixed